### ISSUE TITLE
IPsec: copy-feature in GUI is broken

### ISSUE THREAD

**Issue by fraenki** [opnsense/core#268]
OPNsense 15.7.3

I've setup an IPsec Phase 1 and configured a Phase 2 tunnel. I had to create 4 Phase 2 tunnels with almost the same configuration, so I was using the button _"add a new Phase 2 based on this one"_ to copy the already existing Phase 2 tunnel.

After configuring all 4 Phase 2 tunnels I've noticed that only 2 of them would come up... after fiddling around I came across this post on the pfSense forum: https://forum.pfsense.org/index.php?topic=89044.msg497760#msg497760

Long story short, it seems that the _"add a new Phase 2 based on this one"_ feature/button is somewhat broken.

I deleted all Phase 2 tunnels and recreated them from scratch. Now all tunnels are up.

**Comment by AdSchellevis** [comment]
Hi Frank,
For some reason I couldn't reproduce the issue. Did you perhaps saved a copy of /usr/local/etc/ipsec.conf from both situations? 

Either we accidentally fixed it or some weirdness is going on here, both config.xml sections look alike on my current development box.

**Comment by fraenki** [comment]
Hm, unfortunately I don't have a copy of the original ipsec.conf. It's a rather weird issue that never happened again (maybe because I don't use the "add a new..." button anymore). I'm OK with closing this issue until someone with the same problem comes along.

**Comment by fichtner** [comment]
ok :D
